charts/kafka-connect/templates/deployment.yml

```
@@ -67,6 +67,9 @@ spec:
           volumeMounts:
             {{- tpl (toYaml .) $ | nindent 12 }}
           {{- end }}
+          {{- if .Values.sidecars }}
+          {{- toYaml .Values.sidecars | nindent 8 }}
```

I believe the indentation are not correct.

```suggestion
          {{- toYaml .Values.sidecars | nindent 10 }}
```